sdcclient/_monitor.py

```diff
@@ -754,6 +748,52 @@ def get_metrics(self):
         res = requests.get(self.url + '/api/data/metrics', headers=self.hdrs, verify=self.ssl_verify)
         return self._request_result(res)
 
+    def convert_scope_string_to_expression(self, scope):
+        if scope != None:
```

Comment:
I would prefer:
```python
if scope is None:
    return [True, None]
```

In fact I would return `[True, []]` so we return always the same type. `[]` is also `False` when in conditions.